Examples/JS/flow/ImageType.js

```diff
@@ -0,0 +1,19 @@
+//
+//  ImageType.js
+//  Autogenerated by plank
+//
+//  DO NOT EDIT - EDITS WILL BE OVERWRITTEN
+//  @generated
+//
+//  @flow
```

Comment:
`// @flow` tells Flow to typecheck the file. Otherwise it will ignore it and assume everything coming out of it is `any`.